### DISCUSSION TITLE
Error en el plugin woocommerce

### DISCUSSION THREAD

**Discussion by dairaave** [decidir/sdk-php-v2#32] [General]
Hola! Quería saber si usando el plugin de woocommerce también se debe realizar la integración con código 🤔 ya que al querer realizar test con las credenciales de prueba me arroja "error de pago" 

Al plugin lo agregue manualmente ya que al querer subir el archivo zip que descargue desde la app store de payway me sale "No se ha podido descomprimir el paquete. No sé encontraron plugins"

Ya seguí todos los pasos de la integración que figura en el manual de woocommerce y nada 😭

**Comment by lcorna** [comment]
> Hola! Quería saber si usando el plugin de woocommerce también se debe realizar la integración con código 🤔 ya que al querer realizar test con las credenciales de prueba me arroja "error de pago"
> 
> Al plugin lo agregue manualmente ya que al querer subir el archivo zip que descargue desde la app store de payway me sale "No se ha podido descomprimir el paquete. No sé encontraron plugins"
> 
> Ya seguí todos los pasos de la integración que figura en el manual de woocommerce y nada 😭

te fijaste en los logs que es lo que te tira
